iai-callgrind-runner/src/api.rs

```diff
@@ -365,11 +365,23 @@ impl EventKind {
             _ => None,
         }
     }
+
+    pub fn to_name(&self) -> impl Display + '_ {
```

Comment:
I'm not sure it needs to return a `String` since we only use it as part of a larger `format!`, I thought I would leave it as `impl Display` in case we wanted to switch to a more optimal implementation later.

Reply:
No, it's fine. You can simply use a `String` which implements a lot more traits than just `Display`.